node/libs/roles/src/validator/keys/aggregate_signature.rs

```diff
@@ -1,18 +1,23 @@
-use super::{Error, PublicKey, Signature};
-use crate::validator::messages::{Msg, MsgHash};
 use std::fmt;
-use zksync_consensus_crypto::{bn254, ByteFmt, Text, TextFmt};
+
+use zksync_consensus_crypto::{bls12_381, ByteFmt, Text, TextFmt};
 use zksync_consensus_utils::enum_util::Variant;
 
+use crate::validator::messages::{Msg, MsgHash};
+
+use super::{Error, PublicKey, Signature};
```

Comment:
We conventionally place `use`s in a single block; you can run the following command to achieve order them:

```shell
cargo fmt -- --config imports_granularity=Crate --config group_imports=One
```

@brunoffranca @pompon0 I think this was previously mentioned in `style.md`; or am I misremembering things?

Reply:
yes, it used to be there